src/routes/jira/jira-get.ts

```diff
@@ -49,7 +49,7 @@ const mapSyncStatus = (syncStatus: SyncStatus = SyncStatus.PENDING): string => {
 	}
 };
 
-export const getInstallations = async (subscriptions: Subscription[], log: Logger, gitHubAppId?: number): Promise<InstallationResults> => {
+export const getInstallations = async (subscriptions: Subscription[], log: Logger, gitHubAppId: number | undefined): Promise<InstallationResults> => {
```

Comment:
`gitHubAppId` should be before logger

Reply:
The refactor on this arg position, in this case, is not purpose of this PR, in this line, it merely change the `?`

Reply:
The reason for this PR is to make something mandatory, as such, I'd expect you'd use the proper coding conventions when doing so.

Reply:
fixed. please review.